Tests/Fuzzers/FuzzTarget.cpp

```diff
@@ -51,15 +51,19 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
 	if (!reader.open())
 	{
 		std::cerr << "Error opening the pcap file\n";
-		return -1;
+		return 0;
 	}
 
+	for (size_t i = 0; i < Size; i++)
+		printf("0x%02X ", Data[i]);
+	printf("\n\n");
```

Comment:
I guess it's for debug purposes? Can we remove it?

Reply:
done